src/factory/PlayerbotFactory.h

```diff
@@ -119,7 +119,7 @@ class PlayerbotFactory
     void InitInventorySkill();
     Item* StoreItem(uint32 itemId, uint32 count);
     void InitGuild();
-    void InitArenaTeam();
+    //void InitArenaTeam();
```

Comment:
If this is not to be used anymore, delete it. There shouldn't be commented out code that isn't used anymore, it just adds bloat.